ERCS/erc-8017.md

```diff
@@ -0,0 +1,90 @@
+---
+eip: 8017
```

Comment:
```suggestion
eip: 7593
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
thank you sir, I have committed a new one